### PR TITLE
[#725] Added hover text on links in news page

### LINKED ISSUE #725: Hover text on links to Twitter in news pages
In the news pages, the link to Twitter should describe itself more clearly, hinting it's a link to share the current news on Twitter, for example displaying "Share on twitter" on mouse hover or next to the icon.

Peek 30-09-2020 12-55

### PR DESCRIPTION
**Pull request by ameybhavsar24**
Shows small text when hovered on links to describe the action.
## Description

As already mentioned in #725 , this PR tackles:
* Adds hover text for links in the news page.

## Checklist
- [x] Ready for review! :rocket:

## Fixes

- Fixes #725

### CODE REVIEW

**Review by bfabio — CHANGES_REQUESTED** [review]
Please note the message should be localized Italian and English. You can take a look at `_data/l10n.yml` and use `site.data.l10n[active_lang].t.your_new_message_key` in the templating engine.

The Italian message for "Share on Twitter" is "Condividi su Twitter".

**Comment by ameybhavsar24** [comment]
Ok, I'll see that.

**Comment by ameybhavsar24** [comment]
I have successfully localised the message.
Resolving merge conflicts now.

**Comment by ameybhavsar24** [comment]
I have resolved the conflicts now. @bfabio Can you review?

**Review by bfabio — APPROVED** [review]
Great :+1:

File: _data/l10n.yml [hunk 1]
```diff
@@ -77,7 +77,9 @@ it:
 
     home_tools_title: Strumenti
     home_tools_subtitle: Ci appoggiamo a strumenti già esistenti e molto conosciuti per fornirti le informazioni, il codice e il supporto di cui hai bisogno
-
+    
+    postarticle_socials_icon_title_prefix: Condividi su
```
Comment (sebbalex) [inline _data/l10n.yml:81]: maybe there is no need to create this, we could do as we did here

https://github.com/italia/developers.italia.it/blob/master/_includes/share.html#L3

File: _data/l10n.yml [hunk 2]
```diff
@@ -311,7 +313,9 @@ en:
 
     home_tools_title: Tools
     home_tools_subtitle: We use existing and well-known tools to give you all information, source code and support that you need
-
+    
+    postarticle_socials_icon_title_prefix: Share on
```
Comment (sebbalex) [inline _data/l10n.yml:317]: same as above

**Review by sebbalex — CHANGES_REQUESTED** [review]

File: _data/l10n.yml (hunk 1 above)
Comment (bfabio) [reply _data/l10n.yml:81]: @sebbalex Nice catch!

Comment (ameybhavsar24) [reply _data/l10n.yml:81]: @sebbalex I guess something like
`{{ t.share_msg | capitalize }} {{ t.on_ }} {{ social[0] | capitalize }}`
should render to `Share on Twitter`.
I'll make the changes.

**Comment by ameybhavsar24** [comment]
@sebbalex 
Removed `postarticle_socials_icon_title_prefix` key from _data/l10n.yml and changed social title message to use existing keys
All good?

**Review by sebbalex — APPROVED** [review]
> @sebbalex
> Removed `postarticle_socials_icon_title_prefix` key from _data/l10n.yml and changed social title message to use existing keys
> All good?

@ameybhavsar24 of course, it looks good, thanks for your work!

**Pull request merged**